gradle/java.gradle

```diff
@@ -274,4 +275,8 @@ tasks.withType(Test).configureEach {
   // Disable all tests if current JVM doesn't match version requirements
   // Disable all tests if skipTests property was specified
   enabled = isJavaVersionAllowed(JavaVersion.current()) && !project.rootProject.hasProperty("skipTests")
+
+  retry {
```

Comment:
Please add a big warning "we still need to fix flaky tests not just retry them forever!"